source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -0,0 +1,179 @@
+---
+title: IPv6 subnets on ovirt-provider-ovn
+category: feature
+authors: mdbarroso
+feature_name: Support IPv6 subnets on ovirt-provider-ovn
+feature_modules: ovirt-provider-ovn
+feature_status: In Development
+---
+
+# IPv6 subnets on ovirt-provider-ovn
+
+## Summary
+
+This feature provides support for IPv6 addresses on guest virtual machines
+whose networking is provided by OVN.
+
+Address auto-configuration is - despite desirable - not a must at this stage,
+and its implementation will take place at a later stage.
+
+Related patches can be found on [gerrit topic:provider_ipv6_subnets](https://gerrit.ovirt.org/#/q/topic:provider_ipv6_subnets).
+
+### Owner
+* Feature Owner: [Miguel Duarte Barroso](https://github.com/maiqueb) (mbarroso in #ovirt)
+* Email: <mdbarroso@redhat.com>
+
+### Terminology
+
+* Networking API: [OpenStack RESTful Networking API](https://developer.openstack.org/api-ref/network/v2/)
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that
+brings Software Defined Networking to OVS.
+* oVirt OVN provider: a proxy that implements a subset of the Networking API
+and interacts with OVN.
+* Subnet: a layer 3 concept, created on top of a network. As per openstack's documentation it is a 'block of IP addresses and associated configuration state. Subnets are used to allocate IP addresses when new ports are created on a network.'
```

Comment:
there is a ' in the end of the sentence.
maybe you meant: 'block of IP addresses and associated configuration state'?

Reply:
No, that's the full quote => 'block of IP addresses and associated configuration state. Subnets are used to allocate IP addresses when new ports are created on a network.'

Check [this](https://docs.openstack.org/python-openstackclient/pike/cli/command-objects/subnet.html) for the source.

Reply:
You're absolutely right I did not see it